specification/2.0/README.md

```diff
@@ -4001,7 +4002,17 @@ When the sampler targets a node's rotation property, the resulting ***p***(*t*)
 
 > **Implementation Note:** The first in-tangent ***a***<sub>1</sub> and last out-tangent ***b***<sub>*n*</sub> should be zeros as they are not used in the spline calculations.
 
-# Appendix D: Full Khronos Copyright Statement
+# Appendix D: Image-based Lighting
+
```

Comment:
I think we should consider putting this as normative in an extension while at the same time adding details with regards to how to handle specular and diffuse contribution (eg using spherical harmonics)
This is needed by the certification group in order to get consistent handling of the light setup.

- if that is not an option then I think it should be marked as non-normative (just like the BRDF appendix).